.changeset/new-gifts-thank.md

```diff
@@ -0,0 +1,13 @@
+---
+"@logto/core": patch
```

Comment:
this is not patch? since we have new features added

Reply:
The changeset will add when this feature is full implemented.